Kickstarter-iOS/Features/Comments/Controllers/CommentRepliesViewController.swift

```diff
@@ -169,26 +169,28 @@ final class CommentRepliesViewController: UITableViewController, MessageBannerVi
         }
       }
 
-    self.viewModel.outputs.userBlocked
+    self.viewModel.outputs.didBlockUser
       .observeForUI()
-      .observeValues { [weak self] success in
-        guard let self else { return }
-        self.commentComposer.isHidden = true
-
-        if success {
-          self.messageBannerViewController?
-            .showBanner(with: .success, message: Strings.Block_user_success())
-          self.delegate?.commentRepliesViewControllerDidBlockUser(self)
-        } else {
-          self.messageBannerViewController?
-            .showBanner(with: .error, message: Strings.Block_user_fail())
-        }
+      .observeValues { [weak self] _ in
+        guard let self, let messageBanner = self.messageBannerViewController else { return }
+
+        messageBanner.showBanner(with: .success, message: Strings.Block_user_success())
```

Comment:
It looks like you removed the line `self.commentComposer.isHidden = true` in this refactoring, as well as `self.delegate?.commentRepliesViewControllerDidBlockUser(self)`. Were these deletions intentional?